### DISCUSSION TITLE
B450 GAMING PRO CARBON AC support?

### DISCUSSION THREAD

**Discussion by WhatMeansGithub** [ixjf/MSIRGB#176] [Ideas]
Hi,

I was wondering if this software is able to support this motherboard? It didn't seem to pick mine up. I'm currently pulling hairs trying to get proper ARGB support. I bought a Coolermaster ARGB controller which is also utter crap. Is it possible to somehow add in support for this board?

**Comment by ixjf** [comment]
Possible, maybe. Does Mystic Light work with your motherboard?

**Reply by lightvortex** [reply]
yes

**Reply by ixjf** [reply]
I think this motherboard works in a different way, so I'm not too keen on adding support for it (can't test it, probably a lot of work anyway).
